python/runtime/cudaq/algorithms/py_state.cpp

```diff
@@ -321,6 +418,22 @@ index pair.
   densityMatrix = cudaq.get_state(kernel)
   # Return the upper-left most entry of the matrix.
   value = densityMatrix[0,0])#")
+      .def(
+          "amplitude",
+          [](state &s, std::vector<int> basisState) {
+            return s.amplitude(basisState);
+          },
+          R"#(Return the amplitude of a state in computational basis.
+          
+.. code-block:: python
+
+  # Example:
+  import numpy as np
+
+  # Create a simulation state.
+  state = cudaq.get_state(kernel)
+  # Return the amplitude of |0101>, assuming this is a 4-qubit state.
+  amplitude = state.amplitude([0,1,0,1])#")
```

Comment:
Why not go with a string here that is converted to the appropriate integer index/indices during submission? I find this a bit awkward to have to put commas for a large number of qubits (bitvector would also be a natural choice).